src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/TemporaryEntity.java

```diff
@@ -0,0 +1,81 @@
+package org.openstreetmap.atlas.geography.atlas.raw.slicing;
+
+import java.io.Serializable;
+import java.util.Map;
+
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+/**
+ * Base class for a temporary object, that keeps track of the bare minimum information required to
+ * create some Atlas entity.
+ *
+ * @author mgostintsev
+ */
+public class TemporaryEntity implements Serializable
```

Comment:
Might be nice to make this one abstract, and to add an abstract method `toString` so the sub classes have to implement it.

Reply:
Good idea!